snap/snapcraft.yaml

```diff
@@ -69,6 +69,9 @@ apps:
       - network
       - network-bind
       - norduser-autostart
+      - desktop-legacy
+      - desktop
+      - unity7
```

Comment:
shouldn't this also include X11 and Wayland? 
https://snapcraft.io/docs/desktop-interfaces
```the x11 and wayland (and mir) interfaces are broken out from the desktop interface so developers may specify the environments where the application is expected to work```

Reply:
Turns out we are fine with just `dekstop-legacy`. I tested on Ubuntu/Gnome which uses wyland.